client/packages/common/src/intl/locales/fr/common.json

```diff
@@ -1742,6 +1742,7 @@
   "reason.stored": "Stocké",
   "reason.unknown": "Inconnu",
   "replenishment": "Approvisionnement",
+  "report-code.pending-encounters": "Suivi des visites",
```

Comment:
There's an [existing translation](https://github.com/msupply-foundation/open-msupply/blob/v2.7.0-RC/client/packages/common/src/intl/locales/fr/common.json#L1817) for this. I'm wondering why it doesn't show in this PR?

can you move this to fr-DJ please? the request came through from Samira and there's some question about the translation made.

Reply:
cause it's been changed to `report-code` instead of `report` in [this issue](https://github.com/msupply-foundation/open-msupply/pull/7268) 🙄